### DISCUSSION TITLE
Horizontal display

### DISCUSSION THREAD

**Discussion by giovi321** [HASwitchPlate/HASPone#155] [Ideas]
Since I have already adapted one HAswitchplate to european standard box, it would be great to do so again. However, I remember it was a big pain in the ass starting to write the Nextion file from scratch. Is there a way to roteate 90° the already prepared hmi?
Thanks!

**Comment by aderusha** [comment]
Yes and no.  Yes you can rotate the project, but none of the objects will be automatically aligned correctly so you will need to modify every button on every page.  Additionally, any blueprint which relies on icon placement is not going to work correctly as the x,y coordinates will be different.

So - possible, not easy, and you will probably run into some problems down the road with blueprints that rely on the ability to position elements at a specific x,y coordinate.

edit: [most of what you read here](https://github.com/HASwitchPlate/HASPone/wiki/Modifying-HASPone-to-work-with-other-Nextion-screens) will apply to modifying the project for landscape use.

**Reply by giovi321** [reply]
Ok I'll stick to resketching the display :)
